string id
int64 job_id
int64 object_id
int64 time_start    # time in seconds since Unix epoch
int64 time_end      # time in seconds since Unix epoch
float64 x_center
float64 y_center
float64 yaw         # angle in radians
float64 x1_intersection
float64 y1_intersection
float64 x2_intersection
float64 y2_intersection